### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by nyyshaaa**
1) Videos listing by category and subcategory.
2) will fix css later and auth yet to be added fully.
3) categories and subcategories change section is at bottom.
4) will add reducer later,at present context only.

https://deploy-preview-2--vocal-torrone-b72432.netlify.app/

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,8 +1,21 @@
...
 function App() {
   return (
     <div >
+      <Routes>
+        <Route path="/" element={<Home/>}></Route>
+        <Route path="/video-listing" element={<VideoListing/>}></Route>
+        <Route path="/login" element={<Login/>}></Route>
+        <Route path="/signup" element={<Signup/>}></Route>
+        <Route path="/mock" element={<Mockman/>}></Route>
+      </Routes>
```
Comment (sonishreyas) [inline src/App.jsx:18]: you can just add a /> instead of </Route>

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import HomeStyles from "./Home.module.css";
+import { useVideos } from "../../context/VideosContest";
+import { BottomNav } from "../../components/BottomNav/BottomNav";
+import { GenreSlider } from "../../components/GenreSlider/GenreSlider";
```
Comment (sonishreyas) [inline src/pages/Home/Home.jsx:4]: You can use absolute imports it eliminates all the ../

**Review by sonishreyas — APPROVED** [review]
Code looks good. I had a couple of suggestions which you can implement, rest looks good

File: src/App.jsx (hunk 1 above)
Comment (nyyshaaa) [reply src/App.jsx:18]: okeiy will do it.thanks for suggetion.

File: src/pages/Home/Home.jsx (hunk 2 above)
Comment (nyyshaaa) [reply src/pages/Home/Home.jsx:4]: okeiy will try later,actually i tried in last project but then i got some problem in all of the imports like and getting errors so i reverted the imports.
will read about it and then implement.
thanks for review!

**Pull request merged**